python/lsst/ts/fbs/utils/auxtel/surveys.py

```diff
@@ -104,18 +143,111 @@ def generate_image_survey(
         survey_name=f"{target.survey_name}",
         target_name=f"{target.target_name}",
-        science_program=f"{target.survey_name}",
-        scheduler_note=f"{target.survey_name}:{target.target_name}",
+        science_program=f"{target.science_program}",
+        scheduler_note=f"{target.survey_name}",
+        nside=nside,
+        detailers=survey_detailers,
+    )
+    # Weight all of the basis functions up or down to weight overall survey
+    image_survey.basis_weights *= target.reward_value
+
+    return image_survey
+
+
+def generate_image_survey_from_target(
+    nside: int,
+    target: Target,
+    wind_speed_maximum: float,
+    survey_detailers: typing.List[BaseDetailer],
+    avoid_wind: bool = True,
+    include_slew: bool = True,
+) -> BaseSurvey:
+    """Generate image survey, for single Target with dithers
+
+    Parameters
+    ----------
+    nside : `int`
+        Healpix map resolution.
+    target : `Target`
+        Target for the image survey.
+    wind_speed_maximum : `float`
+        Wind speed limit, in m/s.
+    survey_detailers : `list` of `detailers.BaseDetailer`
+        List of survey detailers.
+    avoid_wind : `bool`, optional
+        If True, add the wind avoidance basis function.
+        If False, drop basis function entirely.
+        Makes use align with the spectroscopic survey.
+    include_slew : `bool`, optional
+        If True, include slewtime basis functions.
+
+    Returns
+    -------
+    image_survey : `FieldSurvey`
+        Image survey.
+    """
+
+    basis_functions = get_basis_functions_image_survey(
+        ra=target.ra.to(unit=u.deg).value,
         nside=nside,
-        detailers=[
-            TrackingInfoDetailer(
-                target_name=f"{target.target_name}",
-                science_program=f"{target.survey_name}",
-            )
-        ]
-        + survey_detailers,
+        note=target.target_name,
+        note_interest=None,
+        ha_limits=target.hour_angle_limit,
+        wind_speed_maximum=wind_speed_maximum,
+        nobs_reference=0,
+        nobs_survey=0,
+        filter_names=target.filters,
+        gap_min=target.visit_gap,
+        additional_notes=None,
+        include_slew=include_slew,
     )
 
+    # Set up a sequence of nexp exposures per filter
+    # sequence_len = target.nexp * len(target.filters)
+    # sequence = ObservationArray(n=sequence_len)
+    # sequence["RA"] = target.ra.to(u.rad).value
+    # sequence["dec"] = target.dec.to(u.rad).value
+    # sequence["exptime"] = target.exptime
+    # sequence["nexp"] = 1
+    # sequence["scheduler_note"] = f"{target.survey_name}"
+    # sequence["target_name"] = f"{target.target_name}"
+    # sequence["science_program"] = f"{target.science_program}"
+    # i = 0
+    # for filter_obs in target.filters:
+    #     sequence["filter"][i : i + target.nexp] = filter_obs
+    #     i += target.nexp
+
+    # Use this method, until there is an update of rubin_scheduler at summit
```

Comment:
Same comment here as above. Ideally we could this all together, but if the timelines don't work out we need to treat these as separate features (the survey update and the scheduler update).

Reply:
Exactly, thanks!